### ISSUE TITLE
Training failure and missing data set

### ISSUE THREAD

**Issue by Sand-Glass** [hzxie/Stereo-3D-Reconstruction#3]
1. When I trained the Stereo2Voxel，the loss continues to rise.The IoU has been maintained at around 0.04.
![image](https://user-images.githubusercontent.com/96899552/147806657-a7946fae-0541-416a-a1a5-6cecb1e02bc7.png)
Then I used the trained model to test.
Here's the result.
![image](https://user-images.githubusercontent.com/96899552/147806708-30b7e9a0-3a18-4732-aa0d-d4e8d4ec6a89.png)
I don't know why,and need your help.

2. `__C.DATASETS.SHAPENET.PT_CLOUD_PATH         = '/home/hzxie/Datasets/ShapeNet/ShapeNetPoints/%s/%s.npy'`
Could you please provide a download link for this dataset?

**Comment by hzxie** [comment]
Please generate the point cloud with the script 
https://github.com/chenhsuanlin/3D-point-cloud-generation/blob/master/densify/densify.py.

**Comment by LLLYLong** [comment]
Hello, would like to ask how the version of your environment is selected? For example, python, pytorch, cuda.How is the chamfering distance compiled and generated?

**Comment by hzxie** [comment]
@LLLYLong 
The package versions may not be important. Just install the dependencies with

```
pip install -r requirements.txt
```

To compile the chamfer distance, please refer to the section "Train/Test Stereo2Point" in README.

**Comment by hzxie** [comment]
@Sand-Glass @LLLYLong 
Please note that the generated point cloud has been uploaded to Google Drive.
